dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers.test.ts

```diff
@@ -10,15 +19,17 @@ describe('get-utilisation-report-reconciliation-details-by-id.controller helpers
   describe('mapFeeRecordEntityToReconciliationDetailsFeeRecordItem', () => {
     const uploadedReport = UtilisationReportEntityMockBuilder.forStatus('PENDING_RECONCILIATION').build();
 
-    it('maps the fee record entity to the reconciliation details fee record item when the currencies match', () => {
+    it('maps the fee record entity to the reconciliation details fee record item', () => {
```

Comment:
This duplicates some of the assertions in the test below, perhaps we want to just test the base fields that don't have much complexity? I guess here that is the id, facility id, exporter and status

Reply:
I think we can still have one test that tests those four fields that asserts they are mapped over like for like, is there some sort of "object containing" assertion we can do to test just those fields?

The reason for not duplicating is then we have one test in this file fail when the one thing goes wrong, not two, and then we immediately see from the test name of the test that failed exactly what failed, rather than having a rather generic test fail and no understanding from the test name why or what it should actually be doing.

Reply:
I don't really mind too much so will approve even if you leave this as is, but for me a test with a name like this, with this style of assertion for a mapping function should only exist if all the mapping function is doing is mapping each field like for like and doesn't do any other changes no matter the value of the fields.